### ISSUE TITLE
Astro.js `*.astro` file support (html / jsx based language)

### ISSUE THREAD

**Issue by JulianCataldo** [gencer/SCSS-Everywhere#77]
Hello there!

Thanks for this extension, it's a delight to use with HTML file.
I would like to translate this developer experience to Astro js file edition:  
https://astro.build

I tried this:

```jsonc
// ...
  "html-css-class-completion.includeGlobPattern": "**/*.{css,scss,sass,eex,slim,tsx,jsx,haml,svelte,liquid,tpl,latte,php,html,twig,astro}"
// ...
```

But it doesn't work.  
I guess it's because SCSS Everywhere is tied to a file extension / language engine combo ?  
For example you can set these two independently with ESLint, Prettier…

Adding language support for `.astro` might be relatively easy, as it's very close to JSX, what do you think?

Thank you 👋

**Comment by gencer** [comment]
What is the vscode extension you use for astro files?

**Comment by JulianCataldo** [comment]
Yes Astro's got a dedicated VS Code extension, with Typescript awareness, default VS Code formatting support, language support etc. etc.
More or less similar to Volar for Vue.

**Comment by TheElegantCoding** [comment]
This is the [Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) I would like to see support for astro files also
